packages/generator/__tests__/assets/templates/index.ts

```diff
@@ -87,7 +86,9 @@ export const business = {
   労働条件通知書,
 };
 
-export const textType = {
+// These tests are slower, so we allow more time for them to pass
+export const slowerAndTextType = {
+  Aone31553QRコード,
```

Comment:
I know this is now not in the suite with the rest of the barcodes, but it's running time was almost the same as the verticalAlignment ones, so it is also quite slow to run compared to the rest of the barcode ones